examples/s3-copy.hs

```diff
@@ -1,13 +1,13 @@
-{-# LANGUAGE OverloadedLists #-}
+{-# LANGUAGE OverloadedLists   #-}
```

Comment:
Can you undo the stylish haskell changes here? I like to keep pragmas and imports non whitespace dependent.